polaris/polaris/models.py

```diff
@@ -22,7 +25,7 @@ class Asset(TimeStampedModel):
     This defines an Asset, as described in the SEP-24 Info_ endpoint.
     """
 
-    code = models.TextField(validators=[MinLengthValidator(1)], default="USD")
```

Comment:
Why remove this?

Reply:
The column is already `NOT NULL` (`null=False` by default in django), so I felt that the `MinLengthValidator` was hand holding. The anchor would have to explicitly create the asset with an empty string as the `code` since `USD` is the default.